masschallenge/django-accelerator
# [AC-9513] Add Sitetree integration

How to test:

- On the accelerate folder
- Run make migrate
- Check any startup profile pages, you should see the top nav items

Siblings PR: https://github.com/masschallenge/accelerate/pull/3311
